omise-woocommerce.php

```diff
@@ -126,17 +128,20 @@ private function include_classes() {
 		require_once OMISE_WOOCOMMERCE_PLUGIN_PATH . '/includes/events/class-omise-event-charge-capture.php';
 		require_once OMISE_WOOCOMMERCE_PLUGIN_PATH . '/includes/events/class-omise-event-charge-complete.php';
 		require_once OMISE_WOOCOMMERCE_PLUGIN_PATH . '/includes/events/class-omise-event-charge-create.php';
+		require_once OMISE_WOOCOMMERCE_PLUGIN_PATH . '/includes/gateway/abstract-omise-payment-offsite.php';
 		require_once OMISE_WOOCOMMERCE_PLUGIN_PATH . '/includes/gateway/class-omise-payment-alipay.php';
```

Comment:
We now have a list of payment methods - why not use that to automate the `require`s?

Reply:
Just don't see much benefit on doing that for now.

Reply:
Just thinking about not repeating ourselves by effectively writing another list of all the payment methods. Better to have one list so that we just need change that instead of writing more boilerplate code every time a new payment method is added

Reply:
As a context of concern, I don't think there is any better place than this place that should be concerning about what file that needed to be included in the plugin. Unless we implement something similar to `autoload` class, having multiple places to list out files that needed to be included is a bit too much of complexity compare to what it will be achieved.

At least, for now.

Reply:
I wasn't suggesting requiring the files elsewhere - it can still be done here. I just thought it would  be beneficial to use the list of payment methods we already have to do the `require`s here - i.e. loop over the names, transform the name to the file name, require the file

Reply:
okay I see. I understand all you suggested but I just don't see that it will greatly benefit more than the current code considering that adding that loop of code here in this place as well.
I've really considered it. Until the point of time where this suggest is shining out and in needed, can we skip this point? 
(although the PayNow PR will be touching this topic again, I still don't see it is in needed to be implemented)